modules/homepage/src/main/frontend/src/themePage/Navbars/SearchBar.jsx

```diff
@@ -241,6 +311,14 @@ function SearchBar(props) {
                             </Typography>
                             {getElementName(element)}
                           </div>)}
+                          secondary={(<div>
```

Comment:
A `<div>` element provided to the `secondary` prop of a ListItemText will cause the warning:
`Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.`.
There are two possible fixes:
1. Use the `disableTypography` option
```suggestion
                          disableTypography
                          secondary={(<div>
```
2. Switch from using `<div>` to `<React.Fragment>`
Either one will work.

Reply:
Implemented this change in commit `2a86d54` by switching from `<div>` to `<React.Fragment>` :heavy_check_mark: